### PR TITLE
Better CPU usage

### PR DESCRIPTION
**Pull request by KeitIG**
Still not perfect, but we're getting there.

Before: 
screen shot 2018-02-27 at 15 49 19

After:

screen shot 2018-02-27 at 15 34 26

For the record, when removing completely the animation: 

screen shot 2018-02-27 at 15 35 07

### CODE REVIEW

**Comment by KeitIG** [comment]
Seems some work from my `sorting` branch got there, no big deal though 😄

File: src/ui/components/Header/PlayingBar.react.js [hunk 1]
```diff
@@ -31,171 +22,23 @@ export default class PlayingBar extends Component {
...
-            </div>
-            <ProgressBar
-              now={elapsedPercent}
-              onMouseDown={this.jumpAudioTo}
-              onMouseMove={this.showTooltip}
-              onMouseLeave={this.hideTooltip}
-            />
-          </div>
-        </div>
+        <PlayingBarInfos
```
Comment (YurySolovyov) [inline src/ui/components/Header/PlayingBar.react.js]: Why is that plural?

File: src/ui/components/Header/QueueListItem.react.js [hunk 2]
```diff
@@ -80,6 +77,9 @@ export default class QueueListItem extends PureComponent {
             <span className='artist'>{ track.artist }</span> - <span className='album'>{ track.album }</span>
           </div>
         </div>
+        <Button bsSize={'xsmall'} bsStyle={'link'} className='remove' onClick={this.remove}>
```
Comment (YurySolovyov) [inline src/ui/components/Header/QueueListItem.react.js]: can't you just use strings directly here?
```jsx
<Button bsSize="xsmall" bsStyle="link" className='remove' onClick={this.remove}> ...
```

**Comment by YurySolovyov** [comment]
> Seems some work from my sorting branch got there, no big deal though

I'd prefer if it was focused on the _one_ issue TBH

**Comment by KeitIG** [comment]
Fair

**Pull request closed without merging**